include/seqan/seeds/seeds_extension.h

```diff
@@ -796,7 +796,7 @@ extendSeed(Seed<Simple, TConfig> & seed,
     // SEQAN_ASSERT_LT(scoreMismatch(scoringScheme), 0);
     SEQAN_ASSERT_LT(scoreGapOpen(scoringScheme), 0);
     SEQAN_ASSERT_LT(scoreGapExtend(scoringScheme), 0);
-    SEQAN_ASSERT_EQ(scoreGapExtend(scoringScheme), scoreGapOpen(scoringScheme));
+    //SEQAN_ASSERT_EQ(scoreGapExtend(scoringScheme), scoreGapOpen(scoringScheme));
```

Comment:
@h-2  @esiragusa 
I would like to ask you check this.

Because of this assertion, we cannot use different scores for gapExtend and gapOpen. 
This is nonsense. Isn't it?
So I would like to comment this out.

If we have to maintain this we also have to remove assignment#4 which uses Gotoh algorithm.
http://seqan.readthedocs.org/en/master/Tutorial/SeedAndExtend.html

Let me know whether it's OK or not :)